cli/cmd/main.go

```diff
@@ -303,6 +305,16 @@ func main() {
 								Required: false,
 								Usage:    "ISO-8601 timestamp indicating the end of the interval filter",
 							},
+							&cli.StringFlag{
+								Name:     dataFlagTaggedUntagged,
+								Required: false,
+								Usage:    "indicates filtering based on data with any tags or no tags (accepts tagged or untagged)",
+							},
+							&cli.StringSliceFlag{
+								Name:     dataFlagTags,
+								Required: false,
+								Usage:    "indicates filtering based on matching tags",
```

Comment:
I think this usage message should indicate that it's going to return all data matching _any_ of the passed tags vs those matching _all_ of them. I think the default there is non-obvious, so good to be explicit

Reply:
agreed, done